substreams/ethereum-maverick/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "substreams-maverick"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "substreams_maverick"
+crate-type = ["cdylib"]
+
+[dependencies]
+substreams.workspace = true
```

Comment:
We're not using workspace dependencies for integrations anymore, as a bump on the parent version would affect all integrations. We need to fix a value here